docs/security/azure-linux-os-guard.md

```diff
@@ -0,0 +1,534 @@
+
+### Scenario 4: Explore Linux Security Modules 
+
+Azure Linux with OS Guard builds on the principle of immutability and strengthens it with two advanced security layers: IPE (Integrity Policy Enforcement) and SELinux mandatory access control. IPE, pioneered by Microsoft and recently upstreamed in the Linux 6.12 kernel, verifies the integrity and authenticity of all executable code in user space. In OS Guard, IPE ensures that only trusted binaries from dm-verity protected volumes, including container layers, are allowed to run. IPE can run in the following modes: Audit or Enforce. 
+
+| **Mode**    | **What It Means**                                                                 |
+|-------------|-----------------------------------------------------------------------------------|
+| Audit       | Logs integrity violations without blocking execution. Useful for testing and policy tuning before enforcement. |
+| Enforce     | Actively blocks execution of binaries that fail integrity checks. Ensures only trusted code runs. |
+
+*Please note, for Azure Linux with OS Guard Public Preview IPE is in Audit mode, with plans to move to Enforce mode in GA.*
+
+Complementing this, SELinux enforces mandatory access control policies to confine processes and containers to strict security domains, ensuring least privilege and reducing the blast radius of any compromise. 
+
+| **Mode**     | **What It Means**                                                                 |
+|--------------|-----------------------------------------------------------------------------------|
+| Permissive   | Logs policy violations but does not enforce restrictions. Helps identify potential issues without impacting workloads. |
+| Enforcing    | Applies mandatory access control policies, blocking actions that violate SELinux rules. Provides strong runtime containment. |
+| Disabled     | SELinux is turned off; no access control or logging occurs. Not recommended for hardened environments. |
+
+*Please note, for Azure Linux with OS Guard Public Preview SELinux is in Permissive mode, with plans to move to Enforcing mode in GA.*
+
+Together, IPE and SELinux provide defense in depth: preventing execution of tampered or unauthorized code, blocking privilege escalation, and maintaining strong isolation between workloads.
+
+To begin exploring these security modules in more depth, ensure you still have access to the node from Scenario 2 through a privileged container as a debugging pod. 
+
+You will first verify that SELinux is in `permissive` mode on your node. You can do so by running the following command:
+
+```bash
+getenforce
+```
+If you prefer to see a more detailed summary of the SELinux status with information on the policy type, you can do so by running the following command:
+
+```bash
+sestatus
+```
+
+You will now verify that IPE is in `audit` mode on your node. *Note: an output of 0 indicates that IPE is in audit mode.*
```

Comment:
Move "*Note: an output of 0 indicates that IPE is in audit mode.*" down to the portion where they `cat /sys/kernel/security/ipe/enforce`